config/_default/config.yaml

```diff
@@ -10,6 +10,10 @@ languageCode: en
 paginate: 7
 rssLimit: 10
 
+params:
```

Comment:
Please move to `/config/_default/params.yaml`

Reply:
done